pandas/io/packers.py

```diff
@@ -357,6 +357,7 @@ def encode(obj):
                     'klass': obj.__class__.__name__,
                     'axes': data.axes,
                     'blocks': [{'items': data.items.take(b.mgr_locs),
+                                'locs': b.mgr_locs.as_array,
```

Comment:
@jreback Is it better to add `locs` (like it is right now) so that new `msgpack` can be decoded in older versions, or replace `items` with `locs` in `encode()` so that the encoded `msgpack` is smaller?